ntp/syncTime.go

```diff
@@ -76,14 +79,34 @@ func NewSyncTime(ntpConfig config.NTPConfig, syncPeriod time.Duration, customQue
 	} else {
 		queryFunc = customQueryFunc
 	}
+
+	hostIndex := checkNTPHost(ntpConfig, queryFunc)
+	log.Info(fmt.Sprintf("Using NTP server : %s", ntpConfig.Host[hostIndex]))
+
 	s := syncTime{
 		clockOffset: 0,
 		syncPeriod:  syncPeriod,
 		query:       queryFunc,
-		ntpOptions:  NewNTPOptions(ntpConfig)}
+		ntpOptions:  NewNTPOptions(ntpConfig, hostIndex)}
+
 	return &s
 }
 
+func checkNTPHost(ntpConfig config.NTPConfig, customQueryFunc func(options NTPOptions) (*ntp.Response, error)) int {
+	ntpConfig.Timeout = 10 * time.Millisecond
```

Comment:
It is not too short for a timeout with only 10ms?

Reply:
You right. Changed , and now the timeout used is the one from config file.